pack/tic/def_encounter.json

```diff
@@ -0,0 +1,598 @@
+		"back_illustrator": "Frej Agelii",
+		"back_name": "Unfathomable Depths",
+		"clues": 1,
+		"code": "07175a",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 14,
+		"faction_code": "mythos",
+		"illustrator": "Andrew Olson",
+		"name": "Deep One Grotto",
+		"pack_code": "def",
+		"position": 175,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After this location is revealed: Increase its flood level. Place the set-aside purple key on it.\n[free] If investigators at this location control both the red key and the blue key and spend 2 [per_investigator] clues, as a group: Take control of the purple key and read <b>Flashback IX</b> in the Campaign Guide.",
+		"traits": "Cave. Y'ha-nthlei.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
+		"back_illustrator": "Frej Agelii",
+		"back_name": "Unfathomable Depths",
+		"clues": 2,
+		"code": "07175b",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 15,
+		"faction_code": "mythos",
+		"illustrator": "Andrew Olson",
+		"name": "Deep One Grotto",
+		"pack_code": "def",
+		"position": 175,
+		"quantity": 1,
+		"shroud": 2,
+		"text": "<b>Forced</b> - After this location is revealed: Increase its flood level. Place the set-aside purple key on it.\n[free] If investigators at this location control both the green key and the yellow key and spend 3 [per_investigator] clues, as a group: Take control of the purple key and read <b>Flashback IX</b>in the Campaign Guide.",
+		"traits": "Cave. Y'ha-nthlei.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
+		"back_illustrator": "Frej Agelii",
+		"back_name": "Unfathomable Depths",
+		"clues": 1,
+		"code": "07176a",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 16,
+		"faction_code": "mythos",
+		"illustrator": "Stanislav Dikolenko",
+		"name": "Cyclopean Ruins",
+		"pack_code": "def",
+		"position": 176,
+		"quantity": 1,
+		"shroud": 3,
+		"text": "<b>Forced</b> - After this location is revealed: It becomes fully flooded. Place the set-aside white key on it.\n[free] If investigators at this location control both the red key and the yellow key and spend 2 [per_investigator] clues, as a group: Take control of the white key and read <b>Flashback X</b> in the Campaign Guide.",
+		"traits": "Cave. Y'ha-nthlei.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```

Comment:
Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).